templates/js/custom-copilot-rag-azure-ai-search/src/app/customSayCommand.js

```diff
@@ -0,0 +1,71 @@
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Any reason to use this way?
module.exports = { xxx } not workable?

Reply:
Here is just copy from SayCommand.js
looks like it is not required, so I remove it and code still work